jaeger/jaeger_install/rhbjaeger-deploying.adoc

```diff
@@ -0,0 +1,65 @@
+[id="deploying-jaeger"]
+= Deploying Jaeger
+include::modules/ossm-document-attributes.adoc[]
+:context: jaeger-deploying
+
+toc::[]
+
+The Jaeger Operator includes a custom resource definition (CRD) file that determines the architecture and configuration settings to use when creating the Jaeger custom resources (CR). You can either install the default configuration or modify the custom resource file to better suit your business requirements.
+
+Each Jaeger instance is associated with a deployment strategy. You define the strategy in a custom resource file and when you create a Jaeger instance the Operator uses the deployment strategy and configuration options in the CR file to create the objects necessary for the deployment.
+
+.Jaeger custom resource file showing deployment strategy
+[source,yaml]
+----
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: simple-prod
+spec:
+  strategy: production <1>
+----
+
+
+<1> The Jaeger Operator currently supports the following deployment strategies:
+
+* *allInOne* (Default) strategy - This strategy is intended for development, testing, and demo purposes. The main backend components, Agent, Collector and Query service, are all packaged into a single executable which is configured (by default) to use in-memory storage.
++
+[NOTE]
+====
+In-memory storage is not persistent, which means that if the Jaeger instance shuts down, restarts, or is replaced, that your trace data will be lost.  For persistent storage, use the production Elasticsearch deployment strategy.
```

Comment:
"use the production or streaming strategies, which use Elasticsearch as the default storage" ?